cmd/installer/cli/join.go

```
@@ -459,9 +462,26 @@ func applyJoinConfigurationOverrides(jcmd *kotsadm.JoinCommandResponse) error {
 	return nil
 }
 
+func getFirstDefinedProfile() string {
```

this should return an error and we should handle this error